relay-event-schema/src/protocol/span.rs

```diff
@@ -257,6 +257,10 @@ pub struct SpanData {
     #[metastructure(field = "thread.name")]
     pub thread_name: Annotated<Value>,
 
+    /// Origin Transaction name of the span.
+    /// For INP spans, this is the route name where the interaction occurred.
```

Comment:
nit: the first line should be separted from the rest, according to [Relay code guidelines](https://www.notion.so/sentry/Relay-Code-and-Architecture-4fac481cf27741aba54d39594f1cae58?pvs=4#987c0416a9f54674999eed59eecbf846):
```suggestion
    /// Origin Transaction name of the span.
    ///
    /// For INP spans, this is the route name where the interaction occurred.
```